src/bin/CMakeLists.txt

```diff
@@ -34,9 +34,19 @@ set(PUBLIC_HEADERS
     ../inc/msquic_linux.h
     ../inc/quic_sal_stub.h)
 
-#install(TARGETS msquic EXPORT msquic DESTINATION "${main_lib_dest}")
-#install(FILES ${PUBLIC_HEADERS} DESTINATION "${include_dest}")
+install(TARGETS msquic EXPORT msquic DESTINATION "${main_lib_dest}")
+install(FILES ${PUBLIC_HEADERS} DESTINATION "${include_dest}")
 
-#configure_file(msquic-config.cmake.in ${CMAKE_BINARY_DIR}/msquic-config.cmake )
-#install(FILES ${CMAKE_BINARY_DIR}/msquic-config.cmake DESTINATION ${msquic_dest})
-#install(EXPORT msquic DESTINATION ${msquic_dest})
+configure_file(msquic-config.cmake.in ${CMAKE_BINARY_DIR}/msquic-config.cmake)
+
+install(FILES ${CMAKE_BINARY_DIR}/msquic-config.cmake DESTINATION ${msquic_dest})
+
+install(EXPORT msquic DESTINATION ${msquic_dest})
+
+if(WIN32)
+    add_library(msquic.lttng INTERFACE)
```

Comment:
Why are you doing this on Windows? There is no lttng on Windows.

Reply:
I need a target on windows, since downstream executables use this target to make loading slightly easier. This just creates an empty target, so every downstream use doesn't require an if check.

Reply:
Ok. Though, I expect this will eventually move into clog somehow...